apps/prairielearn/src/components/QuestionContainer.html.ts

```diff
@@ -392,7 +392,9 @@ export function QuestionFooterContent({
                   <button
                     class="btn btn-info question-save disable-on-submit order-2"
                     ${disableSaveButton ? 'disabled' : ''}
-                    ${question.type === 'Freeform' ? html`name="__action" value="save"` : ''}
+                    ${question.type === 'Freeform'
+                      ? html`type="submit" name="__action" value="save"`
+                      : ''}
```

Comment:
Shouldn't this change be done outside of the conditional? I imagine this issue would be a problem in v2 questions as well.

Reply:
I was erring on the side of not touching v2 questions, but I did test this and it works as expected. Done in d62e259.

Putting in writing: this is a slight change in behavior. Consider the file that drives v2 question submissions:

https://github.com/PrairieLearn/PrairieLearn/blob/master/apps/prairielearn/public/localscripts/question.js

Before this change, if for some reason that script errored or failed to load, clicking "Save" or "Save & Grade" would just do nothing. Now, it will submit a form that almost definitely has empty data. I'm not too concerned about that, just calling it out for consideration and posterity.

Reply:
> Before this change, if for some reason that script errored or failed to load, clicking "Save" or "Save & Grade" would just do nothing.

Are you sure? Buttons inside forms are [by default submit buttons](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#type).

Reply:
> Are you sure?

I guess not! Excellent point; that's how everything in this PR worked without `type="submit"` in the first place. This is just about making things explicit, it doesn't change behavior.